styles.css

```diff
@@ -0,0 +1,156 @@
+/* BACKGROUND */
+body {
+  max-width: 100%;
+  height: auto;
+  font-family: Inter, Arial, Helvetica, sans-serif;
+  background: url("./images/background.png") no-repeat fixed center;
+}
+
+/* HEADER */
+
+/* main box */
+.flex-container {
+  width: 100%;
+  height: 100%;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  box-sizing: border-box;
+}
+
+.social-links a {
+  color: #172b4d;
+  text-align: center;
+}
+
+.outer-container {
+  flex-direction: column;
+  margin-top: 2%;
+}
+
+/* main container */
+.main-container {
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+  margin-top: 9%;
+  width: 100%;
+  padding: 0 17%;
+}
+
+h1 {
+  font-weight: 800;
+  font-size: 40px;
+  margin: 0%;
+}
+
+p {
+  text-align: center;
+  padding: 0 17%;
```

Comment:
- [ ] kindly consider removing `17%` padding for the same reasons I mentioned above and try giving it appropriate padding on both sides as in the figma design.